other/materials_designer/specific_examples/heterostructure_high_k_metal_gate_stack.ipynb

```diff
@@ -0,0 +1,507 @@
+{
```

Comment:
<p>We need to add a link to the original tutorial</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Exabyte-io/api-examples/pull/185/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>